src/lib/serviceRoutines/postQueryContext.cpp

```diff
@@ -49,115 +52,57 @@ static char* xmlPayloadClean(const char*  payload, const char* payloadWord)
 
 /* ****************************************************************************
 *
-* postQueryContext -
+* queryForward - 
+*
+* An entity/attribute has been found on some context provider.
+* We need to forward the query request to the context provider, indicated in qcrsP->contextProvider
```

Comment:
Actually, there can't be any plural here.
This function, *queryForward*, forwards only **one** query to **one** context provider.

The plural would be suited from where this function is called, but not from here.

NTC